books/calculus/bake

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :calculus) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+  BakePreface.v1(book: book)
```

Comment:
Add a newline after the `trash` line so it is clear the comment only applies to trash